doc/dev/crimson/crimson.rst

```diff
@@ -2,90 +2,91 @@
-Please note, `ASan`_ is enabled by default if crimson is built from a source
-cloned using git.
+Please note, `ASan`_ is enabled by default if Crimson is built from a source
+cloned using ``git``.
 
 .. _ASan: https://github.com/google/sanitizers/wiki/AddressSanitizer
 
 Testing crimson with cephadm
 ===============================
 
-The Ceph CI/CD pipeline includes ceph container builds with
-crimson-osd subsitituted for ceph-osd.
+The Ceph CI/CD pipeline builds containers with
+``crimson-osd`` subsitituted for ``ceph-osd``.
 
 Once a branch at commit <sha1> has been built and is available in
-shaman, you can deploy it using the cephadm instructions outlined
+``shaman``, you can deploy it using the cephadm instructions outlined
 in :ref:`cephadm` with the following adaptations.
 
-First, while performing the initial bootstrap, use the --image flag to
-use a crimson build rather than a default build:
+First, while performing the initial bootstrap, use the ``--image`` flag to
+use a Crimson build:
 
 .. prompt:: bash #
 
    cephadm --image quay.ceph.io/ceph-ci/ceph:<sha1>-crimson --allow-mismatched-release bootstrap ...
 
-You'll likely need to include the --allow-mismatched-release flag to
+You'll likely need to supply the ``--allow-mismatched-release`` flag to
 use a non-release branch.
 
-Additionally, prior to deploying the osds, you'll need enable crimson
-and default pools to be created as crimson pools (from cephadm shell):
+Additionally, prior to deploying OSDs, you'll need enable Crimson to
+direct the default pools to be created as Crimson pools.  From the cephadm shell run:
 
 .. prompt:: bash #
 
    ceph config set global 'enable_experimental_unrecoverable_data_corrupting_features' crimson
    ceph osd set-allow-crimson --yes-i-really-mean-it
    ceph config set mon osd_pool_default_crimson true
 
-The first command enables the crimson experimental feature.  Crimson
-is highly experimental, and malfunctions up to and including crashes
+The first command enables the ``crimson`` experimental feature.  Crimson
+is highly experimental, and malfunctions including crashes
 and data loss are to be expected.
 
-The second enables the allow_crimson OSDMap flag.  The monitor will
-not allow crimson-osd to boot without that flag.
+The second enables the ``allow_crimson`` OSDMap flag.  The monitor will
+not allow ``crimson-osd`` to boot without that flag.
 
-The last causes pools to be created by default with the crimson flag.
-crimson pools are restricted to operations supported by crimson.
-crimson-osd won't instantiate pgs from non-crimson pools.
+The last causes pools to be created by default with the ``crimson`` flag.
+Crimson pools are restricted to operations supported by Crimson.
+``Crimson-osd`` won't instantiate PGs from non-Crimson pools.
 
 Running Crimson
 ===============
 
-As you might expect, crimson is not featurewise on par with its predecessor yet.
+As you might expect, Crimson is not yet on par with its predecessor yet with
```

Comment:
s/is not yet on par with its predecessor yet/is not yet on par with its predecessor/

Or maybe "Crimson does not yet have as extensive a feature set as does ``ceph-osd``."